### ISSUE TITLE
cssRules failed on Chrome

### ISSUE THREAD

**Issue by rasc-br** [grinat/leaflet-simple-map-screenshoter#4]
Hello!
Any solution for Chrome console log error?

`Error while reading CSS rules from http://fonts.googleapis.com/css?family=Lato:400,700&display=swap SecurityError: Failed to read the 'cssRules' property from 'CSSStyleSheet': Cannot access rules`

I mean, a solution which I don't need to change the plugin's code.
Thank you!

**Comment by grinat** [comment]
@rasc-br need example on https://jsfiddle.net/ or link to repo with sample

**Comment by rasc-br** [comment]
The code that I'm working from is huge, I couldn't break it apart to share on jsfiddle.
But I found the problem. 
I was importing my font in a "wrong" way, outside vue and directly in the index:
`<link href="//fonts.googleapis.com/css?family=Lato:400,700&display=swap" rel="stylesheet">`
When I imported inside vue (app.vue)
`@import url('https://fonts.googleapis.com/css?family=Lato:400,700&display=swap');`
The problem was solved, no more console logs with exceptions about css rules.

Thank you.
